bundles/org.openhab.core/src/main/java/org/openhab/core/util/ColorUtil.java

```diff
@@ -1176,4 +1224,16 @@ public static double xyToKelvin(double[] xy) {
         double n = (xy[0] - 0.3320) / (0.1858 - xy[1]);
         return (437 * Math.pow(n, 3)) + (3601 * Math.pow(n, 2)) + (6861 * n) + 5517;
     }
+
+    private static BigDecimal polynomialFit(BigDecimal x, BigDecimal[] coefficients) {
```

Comment:
Could you please add Javadoc?  We did the same for other private methods as well. The comment below should also be mentioned here.